qemu/tests/qemu_disk_img_convert.py

```diff
@@ -33,6 +36,28 @@ def convert(self, t_params=None):
         self.trash.append(converted)
         return params
 
+    @error_context.context_aware
```

Comment:
kill_vm = no  will help you to save time for test prepare steps.   variants will make your test matrix looks more clearly